### DISCUSSION TITLE
Unable to login to Vercel CLI - SSL Expired i guess

### DISCUSSION THREAD

**Discussion by sibinanthony** [vercel/community#2579] [Bug Report]
Error: Unexpected error: request to https://api.vercel.com/registration?mode=login failed, reason: Client network socket disconnected before secure TLS connection was established

**Comment by swarnava** [comment]
Hi,

Are you using VPN connection or company firewall? Can you try on different network?

**Reply by sibinanthony** [reply]
i am not using VPN or any firewall . SSL expired . you can simply click on the above URL and see

**Reply by swarnava** [reply]
Hi, I can see that the certificate is very much active: https://www.sslshopper.com/ssl-checker.html#hostname=https://api.vercel.com/registration?mode=login
